ide/xmlprotocol.ml

```diff
@@ -871,13 +869,9 @@ let of_feedback msg =
   Element ("feedback", obj @ ["route",route], [id;content])
 
 let to_feedback xml = match xml with
-  | Element ("feedback", ["object","edit";"route",route], [id;content]) -> {
-      id = Edit(to_edit_id id);
-      route = int_of_string route;
-      contents = to_feedback_content content }
   | Element ("feedback", ["object","state";"route",route], [id;content]) -> { 
-      id = State(to_stateid id);
-      route = int_of_string route;
+      id       = to_stateid id;
+      route    = int_of_string route;
```

Comment:
I have the same comment here. And this illustrates one of my argument. Would you not worry about internal indentation, you would have changed one line here instead of two.